### PR TITLE
Add tox-based Integration Testing Support

### PR DESCRIPTION
**Pull request by peterallenwebb**
Resolves: #109 

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [x] I have added tests & descriptions to my models (and macros if applicable)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      # the other adapters as shown in the below example for redshift
+    #   with:
+    #     # redshift
+    #     REDSHIFT_HOST: ${{ vars.REDSHIFT_HOST }}
+    #     REDSHIFT_USER: ${{ vars.REDSHIFT_USER }}
+    #     REDSHIFT_DATABASE: ${{ vars.REDSHIFT_DATABASE }}
+    #     REDSHIFT_SCHEMA: "integration_tests_redshift_${{ github.run_number }}"
+    #     REDSHIFT_PORT: ${{ vars.REDSHIFT_PORT }}
+    #   secrets:
+    #     DBT_ENV_SECRET_REDSHIFT_PASS: ${{ secrets.DBT_ENV_SECRET_REDSHIFT_PASS }}
```
Comment (emmyoop) [inline .github/workflows/ci.yml:33]:
```suggestion
    #     DBT_ENV_SECRET_REDSHIFT_PASS: ${{ secrets.DBT_ENV_SECRET_REDSHIFT_PASS }}

```

File: integration_tests/.env/postgres.env [hunk 2]
```diff
@@ -0,0 +1,6 @@
+POSTGRES_HOST=localhost
+POSTGRES_USER=root
+DBT_ENV_SECRET_POSTGRES_PASS=password
+POSTGRES_PORT=5432
+POSTGRES_DATABASE=audit_helper_test
+POSTGRES_SCHEMA=audit_helper_integration_tests_postgres
```
Comment (emmyoop) [inline integration_tests/.env/postgres.env:6]:
```suggestion
POSTGRES_SCHEMA=audit_helper_integration_tests_postgres

```

File: supported_adapters.env [hunk 3]
```diff
@@ -0,0 +1 @@
+SUPPORTED_ADAPTERS=postgres
```
Comment (emmyoop) [inline supported_adapters.env:1]:
```suggestion
SUPPORTED_ADAPTERS=postgres

```

File: tox.ini [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# Postgres integration tests for centralized dbt testing
+# run dbt commands directly, assumes dbt is already installed in environment
+[testenv:dbt_integration_postgres]
+changedir = integration_tests
+allowlist_externals = 
+    dbt
+skip_install = true
+commands =
+    dbt --version
+    dbt debug --target postgres
```
Comment (emmyoop) [inline tox.ini:24]:
```suggestion
    dbt debug --target postgres

```

**Review by emmyoop — COMMENTED** [review]
Looks to me that this covers everything to get postgres tests working!  Just some nits on adding empty lines at the end of the files.

I'd like to get approval from a repo owner before merging.

**Review by joellabes — APPROVED** [review]
lgtm!
